strawberry/field.py

```diff
@@ -215,7 +215,27 @@ def resolver(source, info, **kwargs):
             check_permission(source, info, **kwargs)
 
         kwargs = convert_args(kwargs, arguments_annotations)
-        result = wrap(source, info, **kwargs)
+
+        # allow to omit info and root arguments, we do this
+        # by inspecting the resolver arguments,
+        # if it asks for self, the source will be passed as first argument
+        # if it asks for root, the source it will be passed as kwarg
+        # if it asks for info, the info will be passed as kwarg
```

Comment:
I would just move the comment above `kwargs` because I was lost for a moment where kwargs was coming from 😅🤦‍♂️